Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -82,12 +80,72 @@ def tap_new
 
             - run: brew test-bot --only-formulae
               if: github.event_name == 'pull_request'
+
+            - name: Upload bottles as artifact
+              if: always() && github.event_name == 'pull_request'
+              uses: actions/upload-artifact@main
+              with:
+                name: bottles
+                path: '*.bottle.*'
+    YAML
+
+    actions_publish = <<~YAML
+      name: brew pr-pull
+      on:
+        workflow_dispatch:
+          inputs:
+            pull_request:
+              description: Pull request number
+              required: true
+      jobs:
+        pr-pull:
+          runs-on: ubuntu-latest
+          steps:
+            - name: Set up Homebrew
+              uses: Homebrew/actions/setup-homebrew@master
+
+            - name: Set up git
+              uses: Homebrew/actions/git-user-config@master
+
+            - name: Pull bottles
+              env:
+                HOMEBREW_GITHUB_API_TOKEN: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
+                PULL_REQUEST: ${{github.event.inputs.pull_request}}
+              run: brew pr-pull --debug --tap=$GITHUB_REPOSITORY $PULL_REQUEST
+
+            - name: Push commits
+              uses: Homebrew/actions/git-try-push@master
+              with:
+                token: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
+
+            - name: Delete branch
```

Comment:
Not sure about this step. Maybe we can implement something in `brew` for deleting the closed pull request branch (or branches), so they do not stack up and tap creators don't have to remove them manually? Thoughts?

Reply:
Is the PR closed by the step above but just branch not deleted? Might just want to print something telling them to enable the GitHub feature to auto-delete merged branches? Maybe could be one of our own actions for this or, yeh, part of Homebrew/brew.

Reply:
We close pull requests via `Closes #` string in commit messages created by `brew pr-pull`. Branches of those pull requests are not removed when this GitHub auto-delete feature is on. I think we need our solution to this. An Action will do fine.

Reply:
Gotcha.